docs/analysis-workflows/dna-open-access-filtering.md

```diff
@@ -1,8 +1,47 @@
 ---
 id: dna-open-access-filtering
-title: Open Access Filtering
-sidebar_label: Open Access Filtering
+title: Open Access Variant Filtering
+sidebar_label: Open Access Variant Filtering
 platform_key: DOCS_DNA_PIPELINE
 ---
 
-![Reminder Banner](/assets/submission/banner-reminder.svg)
+The SNV/InDel variant calls from both Sanger & MuTect2 calling results on ARGO portal are further processed through the Open Access Variant Filtering Workflow to create VCF files that can be downloaded by ordinary users from the open-access tier. For details about the process workflow, please see the latest version of the [ARGO Open Access Variant Filtering Workflow](https://github.com/icgc-argo-workflows/open-access-variant-filtering/releases)
+
+## Background
+
+Although in cancer research it is not thought to be possible to re-identify patients from somatic variants data as these mutations are found in the tumour not in an individual's germline genome. That means somatic variants do not provide identifying information theoretically and barrier-free release of somatic variants data can occur without compromising patient privacy.
+
+However germline variants can be mistakenly identified as somatic ones, these are so called "gerline leaks" because:
+
+- the imperfect variant calling tools
+- sequencing errors
+- low and uneven reads coverage.
+
+Recent study by [Sendorek, D.H. et al. 2018](https://doi.org/10.1186/s12859-018-2046-0) have indicated that both germline leaks in somatic SNVs and the overlap rate between somatic SNVs with germline database are relatively low. Another study by [Meyerson, W. et al.](https://doi.org/10.1186/s12859-020-3508-8) also inferred that among the shared variants between somatic SNVs with germline database, only 4% are potential germline leaks, while the other variants mostly represent true biological variants that just arose independently in both germline and somatic settings.
+
+It was reported by [Elena Rojano, et al.](https://doi.org/10.1093/bib/bby039) that more than 88% of disease-associated variants are in non-coding regions especially in regulatory regions, e.g:
+
+- cis and trans-regulatory elements such as promoters, enhancers
+- transcribed non-coding regions with regulatory roles, such as miRNAs and lncRNAs
+
+In order to make as many somatic variants as possible to be public accessible to the research community by large with minimum risk of potential germline leaks, we have created the various genomic elements according to latest version of GENCODE annotations (e.g., v38) and defined the open access regions to include the genomic regions with most interests accordingly. For details about the definition of open access genomic regions, please refer to [Open-access Regions](https://github.com/icgc-argo/open-access-regions).
```

Comment:
suggested wording, but @lindaxiang does this make sense?

We want to make as many somatic variants as possible publicly accessible to the research community, with minimum risk of potential germline leaks. Therefore, we have taken the various genomic elements, according to latest version of GENCODE annotations (e.g., v38), and have defined the open access regions to include the genomic regions of highest interest. For details about the definition of open access genomic regions, please refer to [Open Access Regions](https://github.com/icgc-argo/open-access-regions).

Reply:
Yes. It makes sense to me. Thank you very much Kim.